### DISCUSSION TITLE
List of predefined values for login

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#6] [Ideas]
The login box is simple, and should stay like that.  But for those who have to deal with many servers, many databases : it can be painful to have to enter all the data at every login.
I'd like to implement an optional settings array of pre-defined login details.
Every entry of the settings array could contain any of the system, server, username and database.  Password would, of course, not be a setting !

**Comment by nitodico** [comment]
There's already a permanent login option that stores the credentials locally

**Comment by LionelLaffineur** [comment]
My goal is to allow defining on server side a set of predefined credentials to show on the login page in a select to help work with different database instances.

**Comment by LionelLaffineur** [comment]
After some thinking, I think I will do it via a plugin.

**Comment by unix4you2** [comment]
Hi there @LionelLaffineur 
You dont need to do that or create a plugin.   From the Adminer's base version you can send user, password and database to the login automatically.
Just set the variables that you want and send them via POST.  I use that in my framework without problems.

Check this lines in my project as an example:
https://github.com/unix4you2/practico/blob/master/core/marco_dev.php#L236

Regards.

**Reply by LionelLaffineur** [reply]
What I wanted to implement already exists : plugin Server Login.
